overrides/kubejs/server_scripts/Recipes/Ae2_Mods.js

```diff
@@ -771,6 +774,52 @@ ServerEvents.recipes(event => {
     .duration(160)
     .EUt(GTValues.VA[GTValues.LV]);
 
+  //level emitters
+  event.recipes.gtceu.assembler(`ae2:level_emitter`)
+    .itemInputs('16x gtceu:fine_annealed_copper_wire')
+    .itemInputs('4x gtceu:steel_plate')
+    .itemInputs('minecraft:redstone_torch')
+    .itemInputs('#gtceu:circuits/lv')
+    .itemOutputs('ae2:level_emitter')
+    .duration(100)
+    .EUt(GTValues.VA[GTValues.LV])
+
+  event.recipes.gtceu.assembler(`ae2:energy_level_emitter`)
+    .itemInputs('16x gtceu:fine_annealed_copper_wire')
+    .itemInputs('4x gtceu:tin_plate')
+    .itemInputs('minecraft:redstone_torch')
+    .itemInputs('#gtceu:circuits/lv')
+    .itemOutputs('ae2:energy_level_emitter')
+    .duration(100)
+    .EUt(GTValues.VA[GTValues.LV])
+
+  event.recipes.gtceu.assembler(`expatternprovider:threshold_level_emitter`)
+    .itemInputs('16x gtceu:fine_annealed_copper_wire')
+    .itemInputs('4x gtceu:aluminium_plate')
+    .itemInputs('minecraft:redstone_torch')
+    .itemInputs('#gtceu:circuits/lv')
+    .itemOutputs('expatternprovider:threshold_level_emitter')
+    .duration(100)
+    .EUt(GTValues.VA[GTValues.LV])
+  
+  //merequesters
```

Comment:
Move HV Grade materials to EV crate, as well as Voltage to EV for the requester.